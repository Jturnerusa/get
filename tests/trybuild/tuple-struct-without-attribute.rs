use get::Get;

#[derive(Get)]
pub struct NewType(u64);

pub fn main() {}
