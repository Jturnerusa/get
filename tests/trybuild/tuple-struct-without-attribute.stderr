error: proc-macro derive panicked
 --> tests/trybuild/tuple-struct-without-attribute.rs:3:10
  |
3 | #[derive(Get)]
  |          ^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: "expected attribute on tuple struct field"
